setup.md

```diff
@@ -144,6 +163,77 @@ Then go to your browser and enter:
 http://127.0.0.1:8888?token=<whatever token got displayed in the logs>
 ```
 
+## 5. (Alternative) Manually setup the container for remote debugging
```

Comment:
This looks like a nice addition, although I'll personally stick with the workflow that's been working for me.

Reply:
Ok